tests/e2e/specs/onboarding/step-3-complete-campaign.test.js

```diff
@@ -487,10 +487,10 @@ test.describe( 'Complete your campaign', () => {
 			test.describe( 'With WooCommerce tracking disabled', () => {
 				test.beforeAll( async () => {
 					// Reset the showing status for the "Set up paid ads" section.
-					await page.evaluate( () => window.sessionStorage.clear() );
 					await setupAdsAccountPage.mockAdsAccountIncomplete();
 					await completeCampaign.goto();
 					await completeCampaign.clickSkipPaidAdsCreationButton();
+					await page.evaluate( () => window.sessionStorage.clear() );
```

Comment:
Using `sessionStorage` to conditionally show the campaign section should have been removed by #2533. The code comment "Reset the showing status [...]" and this `window.sessionStorage.clear()` seem to be redundant.

Reply:
I can confirm that removing the line to run window.sessionStorage.clear() has no affect on the tests passing, so I think it can be safely removed. Handled in d92ad5fcf.